Gemfile

```diff
@@ -44,6 +44,7 @@ group :test do
   gem "database_cleaner"
   gem "factory_bot_rails"
   gem "govuk-content-schema-test-helpers"
+  gem "launchy"
```

Comment:
Not used for anything in the code at the moment but just useful when developing Capybara tests as it allows you to open a page in the browser at a specified point in the test so you can e.g. work out why your test is failing. Added it in as we kept using it in development, but if you think we shouldn't muddy the dependencies with something not currently used can take it out.

Reply:
Thanks for explaining @BeckaL 😊 I've not used the gem before, so would have to go look it up to figure out how to use it!

I don't have a strong opinion either way as to whether this is a worthwhile addition to the Gemfile or not. It sounds like it could be if it helped with your development workflow.

However, it might be helpful to include some very basic instructions in the commit message, to document what commands you found useful when developing Capybara tests. Or at least to explain _why_ it was added (as you've already done in this comment thread) – so that it's easy for others to find with a `git blame` in the future.